runner/src/server/index.ts

```diff
@@ -41,7 +41,10 @@ const serverOptions = (): ServerOptions => {
 
   const serverOptions: ServerOptions = {
     debug: { request: [`${config.isDev}`] },
-    port: config.port,
+    port: process.env.PORT || config.port,
```

Comment:
take a look at runner/config/default.js, and runner/config/custom-environment-variables.json. when PORT is an environment variable, it will be available for the config files eg default.js, *.json, so no need to use process.env.PORT

Reply:
Ah missed it ! I will revert this part of changes and push

Reply:
no prob, config code went in recently which is probably why you didn't see it before 😄

Reply:
that makes sense now 😄 , I have rolled back the port config!